panther_driver/src/driver_node.py

```diff
@@ -2,23 +2,28 @@
 
 import math
 
+import yaml
+import canopen
 from numpy import NaN
+
 import rospy
-import canopen
+import tf2_ros
+
 from sensor_msgs.msg import BatteryState
 from sensor_msgs.msg import JointState
 from geometry_msgs.msg import Twist
 from geometry_msgs.msg import Pose
 from geometry_msgs.msg import TransformStamped
 from nav_msgs.msg import Odometry
-import tf2_ros
-from PantherKinematics import PantherKinematics
+
 from ClassicKinematics import PantherClassic
 from MecanumKinematics import PantherMecanum
 from MixedKinematics import PantherMix
-import yaml
 
 
+def sign(x):
+    return bool(x > 0) - bool(x < 0)
```

Comment:
why not use `sign` function from `numpy`?

Reply:
Personally, I try not to use numpy functions as long as the code does not require matrix operations. The reason why numpy was included in that code for NaN definitions is unknown for me.

Reply:
I think that we should avoid reimplementing functions and if numpy is already used, then it won't add an extra dependency